runbotics/runbotics-orchestrator-ui/src/src-landing/components/Navbar/Navbar.tsx

```diff
@@ -36,7 +35,7 @@ const Navbar: FC<NavbarProps> = ({
             <ul className={`${styles.navLinkWrapper} ${isMobileVisible ? styles.minimized : navMobileStyle}`}>
                 <li className={styles.listItem}>
                     <Link
-                        href={`#${BENEFITS_SECTION_ID}`}
+                        href={`/#${BENEFITS_SECTION_ID}`}
```

Comment:
bc we want to use nav on blog page, if we do it without /, it would look like this 'url.com/blog#BENEFITS_SECTION_ID' and not 'url.com/#BENEFITS_SECTION_ID'

Reply:
understandable